sharedb.planx.uk/server.js

```diff
@@ -16,6 +18,10 @@ const sharedb = new ShareDB({
   }),
 });
 
+// Setup JSDOM and DOMPurify
```

Comment:
Docs: https://github.com/cure53/DOMPurify#running-dompurify-on-the-server

Reply:
I hadn't totally connected last week that sharedb qualifies as "on the server" and therefore was trying to use _without_ JSDOM, but this makes much more sense in hindsight :+1: